### PR TITLE
Move data format information out of main README and into `doc/`

### PR DESCRIPTION
**Pull request by jaclyn-taroni**
### Purpose/implementation Section

The main repository README contains a lot of information. Here I'm moving the Data Formats section to its own markdown file in `doc/` to shorten the main README. In addition, I have now broken the Data Formats section up to reflect that we in essence have original data files (PBTA data files) and derived files (analysis files), mirroring the format of `data-description.md`.  I've also changed the ordering of the main README a bit to better reflect (in my opinion) the other changes.

#### What scientific question is your analysis addressing?

No scientific question. This is about contributor friendliness!

#### What GitHub issue does your pull request address?

Related to #334 

### Directions for reviewers. Tell potential reviewers what kind of feedback you are soliciting.

* Is this documentation clear?
* The restructuring of this section is likely to increase the burden on maintainers. Do you have ideas about how to ameliorate that?

#### Reproducibility Checklist

N/A documentation change

### CODE REVIEW

File: doc/data-formats.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+* `pbta-snv-lancet.vep.maf.gz`
+* `pbta-snv-mutect2.vep.maf.gz`
+* `pbta-snv-strelka2.vep.maf.gz`
+* `pbta-snv-vardict.vep.maf.gz`
+
+#### Somatic Copy Number Variant (CNV) Data
+
+Somatic Copy Number Variant (CNV) data are provided in a modified [SEG format](https://software.broadinstitute.org/software/igv/SEG) for each of the [applied software packages](https://alexslemonade.github.io/OpenPBTA-manuscript/#somatic-copy-number-variant-calling) and denoted with the `pbta-cnv` prefix.
+
+  * `pbta-cnv-cnvkit.seg.gz` is the the CNVkit SEG file. This file contains an additional column `copy.num` to denote copy number of each segment, derived from the CNS file output of the algorithm described [here](https://cnvkit.readthedocs.io/en/stable/fileformats.html).
```
Comment (jashapiro) [inline doc/data-formats.md]: I like longer links. 🤷‍♂️
```suggestion
  * `pbta-cnv-cnvkit.seg.gz` is the the CNVkit SEG file. This file contains an additional column `copy.num` to denote copy number of each segment, derived from the CNS file output of the algorithm [described here](https://cnvkit.readthedocs.io/en/stable/fileformats.html).
```

File: doc/data-formats.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+  * `pbta-cnv-controlfreec.tsv.gz` is the ControlFreeC TSV file. It is a merge of `*_CNVs` files produced from the algorithm, and columns are described [here](http://boevalab.inf.ethz.ch/FREEC/tutorial.html#OUTPUT).
```
Comment (jashapiro) [inline doc/data-formats.md]:
```suggestion
  * `pbta-cnv-controlfreec.tsv.gz` is the ControlFreeC TSV file. It is a merge of `*_CNVs` files produced from the algorithm, and columns are [described here](http://boevalab.inf.ethz.ch/FREEC/tutorial.html#OUTPUT).
```

File: doc/data-formats.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+Collapsed expression matrices are products of the [`analyses/collapse-rnaseq`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/collapse-rnaseq) analysis module.
+In cases where more than one Ensembl gene identifier maps to the same gene symbol, the instance of the gene symbol with the maximum mean FPKM in the RSEM FPKM file is retained to produce the following files:
+
+* `pbta-gene-expression-rsem-fpkm-collapsed.polya.rds`  
+* `pbta-gene-expression-rsem-fpkm-collapsed.stranded.rds`
+
+#### Independent Specimen Lists
+
+Independent specimen lists are the products of the [`analyses/independent-samples`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/independent-samples) analysis module.
+For participants with multiple tumor specimens, independent specimen lists are provided as TSV files with columns for participant ID and specimen ID (methods described [here](https://alexslemonade.github.io/OpenPBTA-manuscript/#selection-of-independent-samples)).
```
Comment (jashapiro) [inline doc/data-formats.md]:
```suggestion
For participants with multiple tumor specimens, independent specimen lists are provided as TSV files with columns for participant ID and specimen ID (methods [described here](https://alexslemonade.github.io/OpenPBTA-manuscript/#selection-of-independent-samples)). 
```

File: doc/data-formats.md [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+  * `independent-specimens.wgs.primary-plus.tsv` as above, but including non-primary tumors where a primary tumor sample is not available.
+  * `independent-specimens.wgswxs.primary.tsv` Only primary tumors, but with WXS where WGS is not available. 
+  * `independent-specimens.wgswxs.primary-plus.tsv` as above, but including non-primary tumors where a primary tumor sample is not available.
+
+Note that these independent specimen files do not address the issue of participants with multiple tumor specimens in RNA-seq data at this time.
+
+#### Putative Oncogenic Fusion List
+
+The filtered and prioritized fusion list is a product of the [`analyses/fusion_filtering`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/fusion_filtering) analysis module. 
+The methods are described [here](https://alexslemonade.github.io/OpenPBTA-manuscript/#fusion-prioritization).
```
Comment (jashapiro) [inline doc/data-formats.md]:
```suggestion
The methods are [described here](https://alexslemonade.github.io/OpenPBTA-manuscript/#fusion-prioritization).
```

File: doc/data-formats.md [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+* `pbta-gene-expression-rsem-tpm.polya.rds`
+* `pbta-gene-expression-rsem-tpm.stranded.rds`
+* `pbta-isoform-counts-rsem-expected_count.polya.rds`
+* `pbta-isoform-counts-rsem-expected_count.stranded.rds`
+* `pbta-isoform-expression-rsem-tpm.polya.rds`
+* `pbta-isoform-expression-rsem-tpm.stranded.rds`
+
+See [the data description file](data-description.md) for more information about the individual gene expression files.
+
+If your analysis requires de-duplicated gene symbols as row names, please use the collapsed matrices provided as part of the data download (see [below](#collapsed-expression-matrices)).
```
Comment (jashapiro) [inline doc/data-formats.md]:
```suggestion
If your analysis requires de-duplicated gene symbols as row names, please use the collapsed matrices provided as part of the data download ([see below](#collapsed-expression-matrices)).
```

File: README.md [hunk 6]
```diff
@@ -1,47 +1,45 @@
+# OpenPBTA-analysis
+
+The Open Pediatric Brain Tumor Atlas (OpenPBTA) Project is an effort to describe the landscape of tumors in the [Children's Brain Tumor Tissue Consortium](https://cbttc.org/) and the PNOC003 DIPG clinical trial from the [Pediatric Pacific Neuro-oncology Consortium](http://www.pnoc.us/).
+This is an open analysis effort that is organized on GitHub.
```
[the remark this answers is not in the document]
Comment (jaclyn-taroni) [reply README.md]: I am going to link this in #332. I think this comment should be addressed at the same time as that issue is addressed.

**Comment by jaclyn-taroni** [comment]
Thanks for the comments @jashapiro, ready for another look!

File: README.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (jharenza) [reply README.md]: I can work on this tomorrow.

**Pull request merged**